packages/components/src/internal/components/entities/models.ts

```diff
@@ -510,24 +517,35 @@ export class EntityIdCreationModel extends Record({
         if (this.entityCount > 0) {
             queryInfo.getInsertColumns().forEach(col => {
                 const colName = col.name;
-
-                if (col.isExpInput()) {
+                let selected;
+                if (col.isExpInput() && this.creationType !== SampleCreationType.Aliquots) {
                     // Convert parent values into appropriate column names
                     const sq = EntityIdCreationModel.revertParentInputSchema(col);
 
                     // should be only one parent with the matching schema and query name
-                    const selected = this.entityParents.reduce((found, parentList) => {
+                    selected = this.entityParents.reduce((found, parentList) => {
                         return (
                             found ||
                             parentList.find(parent => parent.schema === sq.schemaName && parent.query === sq.queryName)
                         );
                     }, undefined);
-                    if (selected && selected.value) {
-                        values = values.set(colName, selected.value);
-                        parentCols.push(colName);
-                    }
+
+                }
+                else if (col.isAliquotParent() && this.creationType === SampleCreationType.Aliquots) {
+                    selected = this.entityParents.reduce((found, parentList) => {
+                        return (
+                            found ||
+                            parentList.find(parent => parent.isAliquotParent)
```

Comment:
Is this going to show only those samples that are already aliquot parents?  Don't we want to show all samples since any of them could be aliquot parents?

Reply:
This function is used to match querycolumn to entityParentType. All samples of the target type will be available as aliquot parents, see the lookup def in generateColumn above.